configure

```diff
@@ -833,11 +839,14 @@ def configure_intl(o):
   ]
   def icu_download(path):
     # download ICU, if needed
+    if not os.path.exists(options.download_path):
```

Comment:
Something of an anti-pattern, this.  Whether the path exists doesn't say anything about it being accessible.